integration_tests/sdk/flow_test.py

```diff
@@ -169,3 +170,22 @@ def test_refresh_flow(client):
         wait_for_flow_runs(client, flow.id(), num_runs=num_initial_runs + 1)
     finally:
         client.delete_flow(flow.id())
+
+
+@pytest.mark.publish
+def test_get_flow(client):
```

Comment:
Tiny nit here -- shouldn't this be `test_get_artifact_from_flow` or something similar?